ksqldb-engine/src/test/java/io/confluent/ksql/internal/StorageUtilizationMetricsReporterTest.java

```diff
@@ -54,17 +53,17 @@
 
   @Before
   public void setUp() {
-    listener = new StorageUtilizationMetricsReporter(metrics);
+    listener = new StorageUtilizationMetricsReporter();
+    listener.configure(
+        ImmutableMap.of(
+            KsqlConfig.KSQL_INTERNAL_METRICS_CONFIG, metrics
+        )
+    );
```

Comment:
Rather than creating special constructors for only the tests to use, why not just have the tests call `configure` the same way the production code does?